releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -34,13 +33,12 @@
       "cli",
       "windows"
     ],
-    "duplicate": true,
-    "is_mapped": true
```

Comment:
This was remove, you want to go back in this diff commit and add the `is_mapped": true` values back.